pkgs/development/libraries/pipewire/default.nix

```diff
@@ -170,9 +158,7 @@ let
       "-Dudevrulesdir=lib/udev/rules.d"
       "-Dinstalled_tests=enabled"
       "-Dinstalled_test_prefix=${placeholder "installedTests"}"
-      "-Dpipewire_pulse_prefix=${placeholder "pulse"}"
       "-Dlibjack-path=${placeholder "jack"}/lib"
-      "-Dlibv4l2-path=${placeholder "out"}/lib"
```

Comment:
Did you verify that v4l2 still works the way it is intended?

Reply:
Yes, it just ends up in $out/lib now which is fine: https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/master/pipewire-v4l2/src/meson.build?ref_type=heads#L18